webapp/channels/src/components/actions_menu/actions_menu.tsx

```diff
@@ -387,12 +387,15 @@ export class ActionMenuClass extends React.PureComponent<Props, State> {
                 open={this.props.isMenuOpen}
                 onToggle={this.handleDropdownOpened}
             >
-                <OverlayTrigger
-                    className='hidden-xs'
-                    delayShow={500}
+                <WithTooltip
+                    id={`${this.props.location}_${this.props.post.id}_tooltip`}
+                    title={
+                        <FormattedMessage
+                            id='post_info.tooltip.actions'
+                            defaultMessage='Message actions'
+                        />
+                    }
                     placement='top'
-                    overlay={this.tooltip}
```

Comment:
Since you removed this line, `this.tooltip` can also be removed (around line 115).

Reply:
Updated